### ISSUE TITLE
fbrnch import: Crashing

### ISSUE THREAD

**Issue by farchord** [juhp/fbrnch#40]
I was closing a package review request (kf6-kcodecs). I first did fbrnch request-branch (Which worked) and then fbrnch import. Here's the full log:

`fbrnch import kf6-kcodecs kcodecs/kf6-kcodecs-5.240.0\^20230829.232811.ea56b58-1.fc40.src.rpm

= kf6-kcodecs =
cloning...

https://bugzilla.redhat.com/show_bug.cgi?id=2240693

[Snipped: Bug report text]

Press Enter to continue: 
SRPM URL: https://api.nicehomeservices.com/images/temp/kf6-kcodecs-5.240.0^20230829.232811.ea56b58-1.fc40.src.rpm
fbrnch: Prelude.head: empty list
CallStack (from HasCallStack):
  error, called at libraries/base/GHC/List.hs:1646:3 in base:GHC.List
  errorEmptyList, called at libraries/base/GHC/List.hs:85:11 in base:GHC.List
  badHead, called at libraries/base/GHC/List.hs:91:48 in base:GHC.List
  head, called at src/Cmd/Import.hs:61:21 in main:Cmd.Import
`

**Comment by juhp** [comment]
Hmm this is a little sad/embarrassing...

```
$ ghci
ghci> import Network.URI
ghci> isURI "https://api.nicehomeservices.com/images/temp/kf6-kcodecs-5.240.0^20230829.232811.ea56b58-1.fc40.src.rpm"
False
```

**Comment by juhp** [comment]
So the correct representation would be https://api.nicehomeservices.com/images/temp/kf6-kcodecs-5.240.0%5E20230829.232811.ea56b58-1.fc40.src.rpm

**Comment by juhp** [comment]
I had a fix for this ready - will push it here soon

**Comment by juhp** [comment]
Should be fixed in [1.3.3](https://hackage.haskell.org/package/fbrnch-1.3.3)
